src/Integrations/Greenhouse/SettingsGreenhouse.php

```diff
@@ -324,7 +325,7 @@ public function getSettingsGlobalData(): array
 			[
 				'component' => 'intro',
 				'introTitle' => __('How to get the Job Board name?', 'eightshift-forms'),
-				'introTitleSize' => 'medium',
+				'introTitleSize' => 'small',
 				// phpcs:ignore WordPress.WP.I18n.NoHtmlWrappedStrings
 				'introSubtitle' => __('<ol>
```

Comment:
Translation using `sprintf` to avoid placing HTML in translatable string.

Reply:
this will be refactored in the next release so I would leave it as is for now